include/robot_interfaces/n_joint_robot_types.hpp

```diff
@@ -15,6 +15,9 @@
 #include <robot_interfaces/robot_backend.hpp>
 #include <robot_interfaces/robot_data.hpp>
 #include <robot_interfaces/robot_frontend.hpp>
+#include <robot_interfaces/loggable.hpp>
+
+using namespace std;
```

Comment:
Please do not "use" any namespace in a header file.  The reason is that any file that includes this one will also be affected by this which can result in name collisions.